builders/trailsBuilder.js

```diff
@@ -66,6 +66,38 @@ const queries = [
       WHERE (type = 'hike' OR type = 'horse' OR type = 'bike') AND name != ''
     `
   },
+  {
```

Comment:
Macro feedback:  
- All _SELECT_ statements should have a corresponding _GROUP BY_ statement.  
- Never use _SELECT *_.  It runs much slower than specifying columns to select.